### DISCUSSION TITLE
Question about Splitting the Validation Dataset

### DISCUSSION THREAD

**Discussion by USTCdyf** [SalesforceAIResearch/uni2ts#123] [Q&A]
When I run this to split the validation set
```
python -m uni2ts.data.builder.simple \
    "load" \
    "/home/dyf/code/pilu/dataset/v4/actual_load.csv" \
    --date_offset '2024-04-30 23:45:00'\
    --dataset_type wide

```
I found that in the training set, I have 45,791 data points ( total of 57,599), but in the validation set, I get all 57,599 data points. Could you explain why this is the case?

example:
![QQ_1726221054729](https://github.com/user-attachments/assets/83b9733b-eccd-40ef-abb7-5e37862325d2)
![QQ_1726221187782](https://github.com/user-attachments/assets/033804de-68fe-4ef0-911b-95772bbe9622)
![QQ_1726221206525](https://github.com/user-attachments/assets/4c59817a-6a48-4fd2-822c-f518e363997e)

**Comment by USTCdyf** [comment]
I think I’ve understood the problem: in the data directory, the val_data file includes the full dataset, and then the slicing is handled later through configurations such as the offset. This offset parameter allows for slicing specific parts of the dataset for evaluation or validation. Is it right?
